### ISSUE TITLE
Ubuntu matplotlib TkAgg problems

### ISSUE THREAD

**Issue by breznak** [htm-community/river-runner#6]
FYI This solved my struggle with `--plot` option and matplotlib on ubuntu, 
https://stackoverflow.com/questions/12933335/tkagg-backend-problems

**Comment by rcrowder** [comment]
Nice find! Did you try other back-ends with MPL on Ubuntu (maybe Qtagg)?
GTK was recommended for Windows, but adding plt.pause(0.1) helped event handlers get process time (see https://github.com/nupic-community/river-runner/pull/5).

**Comment by rhyolight** [comment]
@breznak Maybe we should add this to the README or just enable the wiki?

**Comment by breznak** [comment]
@rhyolight ...what suits you better, just it should be mentioned somewhere.
@rcrowder I tried other backends, Qt,GTK,.. had some trouble still.

**Comment by rcrowder** [comment]
@breznak matplotlib needs to be used in a certain why when interactively updating it with data. Adding the millisecond pause is a hack to help background processing. It's still useful at a graphing tool. But then we now have nupic.visualizations and Marcus/Felix work too :)

**Comment by rhyolight** [comment]
Yes, I want to move away from matplotlib and towards some web-base charting as soon as live charts are ready in nupic.visualization.
